### PR TITLE
added black duck policy for cluster management

### PR DESCRIPTION
**Pull request by gautambaghel**
@ycao56 - Please double check this PR

### CODE REVIEW

**Comment by dhaiducek** [comment]
@gautambaghel We've updated the primary branch. Would you please edit this PR to change the base branch from `master` to `main`? Thanks!

**Comment by gautambaghel** [comment]
@dhaiducek -Done!

File: community/SI-System-and-Information-Integrity/policy-blackduck.yaml [hunk 1]
```diff
@@ -0,0 +1,194 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-blackduck
+  namespace: default
```
Comment (ycao56) [inline community/SI-System-and-Information-Integrity/policy-blackduck.yaml]: remove hard coded namespace for policy

File: community/SI-System-and-Information-Integrity/policy-blackduck.yaml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+              metadata:
+                labels:
+                  com.blackducksoftware.com.pod.overall-status: IN_VIOLATION
+                namespace: default
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: binding-blackduck-operator
+  namespace: default
```
Comment (ycao56) [inline community/SI-System-and-Information-Integrity/policy-blackduck.yaml]: remove hard coded namespace for policy

File: community/SI-System-and-Information-Integrity/policy-blackduck.yaml [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+subjects:
+- name: blackduck-operator
+  kind: Policy
+  apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-blackduck-operator
+  namespace: default
```
Comment (ycao56) [inline community/SI-System-and-Information-Integrity/policy-blackduck.yaml]: remove hard coded namespace for policy

**Comment by ycao56** [comment]
@gautambaghel Thanks for the PR! Can you also add this policy to https://github.com/open-cluster-management/policy-collection/blob/main/community/README.md ?
It's also nice to have some descriptions to explain what policy does in policy-blackduck.yaml file itself. see https://github.com/open-cluster-management/policy-collection/blob/main/stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml#L1-L6

**Comment by gautambaghel** [comment]
@ycao56 - made requested changes, pls verify. thanks!

File: community/README.md [hunk 4]
```diff
@@ -11,23 +11,30 @@ Policies in this folder are organized by [NIST Special Publication 800-53](https
...
-- [MA - Maintenance](#maintenance)
-- [MP - Media Protection](#media-protection)
-- [PE - Physical and Environmental Protection](#physical-and-environmental-protection)
-- [PL - Planning](#planning)
-- [PS - Personnel Security](#personnel-security)
-- [RA - Risk Assessment](#risk-assessment)
-- [SA - System and Services Acquisition](#system-and-services-acquisition)
-- [SC - System and Communications Protection](#system-and-communications-protection)
-- [SI - System and Information Integrity](#system-and-information-integrity)
+- [Policies -- Community](#policies----community)
```
Comment (ycao56) [inline community/README.md]: These seems to be unexpected change. Can you revert? Others look fine to me

**Review by ycao56 — CHANGES_REQUESTED** [review]

File: community/README.md (hunk 4 above)
Comment (gautambaghel) [reply community/README.md]: @ycao56 -- Visual studio automatically fixed the table of contents, this looks correct to me. The table of content was inaccurate previously.

If you'd still like me to revert it, I can. Thanks

Comment (ycao56) [reply community/README.md]: @gautambaghel please revert. Thanks.

**Review by ycao56 — APPROVED** [review]

**Pull request merged**